### DISCUSSION TITLE
Isolate forder or layer

### DISCUSSION THREAD

**Discussion by Jebs88** [keiwando/toonsquid#635] [Ideas]
Isolate a layer or a folder like in Adobe Illustrator would be nice. Where you can edit a layer without the problem of accidentally miss clicking the other layers, while still be able to see the other layers in the background.

**Comment by andydjames** [comment]
There is already the "Focus" command in the pop up menu on layers on the Timeline, which brings the layer contents on the current frame into view on the canvas. It doesn't disable other animation or drawing layers. 
If you want to avoid unwanted editing, you can lock animation layers and individual drawing layers though.

**Reply by Jebs88** [reply]
Yes, focus helps me at times. Also made a folder so i can lock the other layers easily, but this would be really good as an update.
